Kitodo/src/main/java/org/kitodo/workflow/model/beans/Diagram.java

```diff
@@ -0,0 +1,136 @@
+/*
+ * (c) Kitodo. Key to digital objects e. V. <contact@kitodo.org>
+ *
+ * This file is part of the Kitodo project.
+ *
+ * It is licensed under GNU General Public License version 3 or later.
+ *
+ * For the full copyright and license information, please read the
+ * GPL3-License.txt file that was distributed with this source code.
+ */
+
+package org.kitodo.workflow.model.beans;
+
+import java.util.Objects;
+
+import org.camunda.bpm.model.bpmn.instance.Process;
+
+public class Diagram {
+
+    private String id;
+    private String title;
+    private String outputName;
+    private Integer docket;
+    private Integer ruleset;
+    private static final String NAMESPACE = "http://www.kitodo.com/template";
```

Comment:
- Why are you using URI `kitodo.com` as the project is using `kitodo.org` domain?
- You are dublicating this constant value in class `KitodoTask` - should this constant with same value not defined on one place and used from both classes?

Reply:
Diagram doesn't extend KitodoTask. Do you think that i would be also better to store in kitodo_config.properties?

Reply:
- If this value should be changed by user / administrator then it belongs to kitodo_config.properties.
- If this value should not be changed from outside, then it should stay here.

So, at least change the URI value to `kitodo.org`.

Reply:
No, basically it shouldn't be changed, as it is defined in modeler.